Ginger/GingerCoreCommon/Repository/RepositoryItemBase.cs

```diff
@@ -72,12 +72,13 @@ public abstract class RepositoryItemBase : INotifyPropertyChanged, ISearchFilter
 
         //DO Not save
         protected ConcurrentDictionary<string, object> mBackupDic;
+        protected bool mBackupInProgress = false;
 
         public bool IsBackupExist
         {
             get
             {
-                if (mBackupDic != null)
+                if (mBackupDic != null && mBackupInProgress == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5970337)